whipper/image/toc.py

```diff
@@ -138,9 +137,9 @@ class TocFile(object):
 
     def __init__(self, path):
         """
-        :type  path: unicode
+        :type  path: str
         """
-        assert isinstance(path, unicode), "%r is not unicode" % path
+        assert isinstance(path, str), "%r is not string" % path
```

Comment:
Same message other places was changed to "is not str". Why "string" here? (I think "string" is probably better, but then it should be "string" everywhere.)

Reply:
Replaced `string` with `str`.